pyomo/contrib/mindtpy/initialization.py

```diff
@@ -61,7 +61,7 @@ def MindtPy_initialize_master(solve_data, config):
         # if config.strategy == 'ECP':
         #     add_ecp_cut(solve_data, config)
```

Comment:
This line here suggests that we were adding the cuts at the initial point for our implementation. This makes sense if we are dealing with a convex problem since at the initialization we can evaluate the variables. Consider uncommenting this line and modifying the code (updating `add_ecp_cut(solve_data, config)` to your new function `add_ecp_cuts(.)`)